.github/pull_request_template.md

```diff
@@ -1,23 +1,69 @@
 <!--
+A few suggestions about filling out this PR
 
-Replace this comment with a description of what's being changed by this PR.
+1. Use a descriptive title for the PR.
+2. If this pull request is work in progress, create a draft PR instead of prefixing the title with WIP.
+3. Please label this PR at least one of the following labels, depending on the scope of your change:
+- type:new-feature, which adds new behavior
+- type:bug fix
+- type:enhancement, which modifies existing behavior
+- type:breaking-change
+4. Remove those recommended/optional sections if you don't need them. Only "What does this PR do" and "Checklist" are mandatory.
+5. Submit the pull request: Push your local changes to your forked copy of the repository and submit a pull request (https://help.github.com/articles/using-pull-requests).
+6. Please be patient. We might not be able to review your code as fast as we would like to, but we'll do our best to dedicate to it the attention it deserves. Your effort is much appreciated!
+-->
 
-If this PR should close an issue, please add one of the magic keywords
-(e.g. Fixes) followed by the issue number. For more info see:
-https://help.github.com/articles/closing-issues-using-keywords/
+## What does this PR do?
+<!-- _(Mandatory)_
+Replace this comment with a description of what's being changed by this PR. Please explain the WHAT: A clear and concise description of what (patterns used, algorithms implemented, design architecture, message processing, etc.)
+-->
 
-If this pull request is work in progress, create a draft PR instead of prefixing the title with WIP.
+## Checklist
+<!-- _(Mandatory)_
+List here all the items you have verified BEFORE sending this PR. Please DO NOT remove any item, striking through those that do not apply. (Just in case, strikethrough uses two tildes. ~~Scratch this.~~)
+-->
+- [ ] I have signed the [Contributor License Agreement](https://www.elastic.co/contributor-agreement/). 
```

Comment:
Uhm, is this required? I mean, the repo itself got a PR check validation for the CLA and if not then it will fail the check. And the link in the GitHub check points to the CLA DOCS

Reply:
I’m a bit torn on this one. It can sometimes be useful as not all contributors have a look at the checks. The checks are usually added by ci after a build is done which may take a while. I think the ideal solution would be to let a bot write a comment if the contributor didn’t sign the cla yet.

Reply:
Do we have that Bot in place?

Reply:
For getting progress in this one, we can remove the item and add it in the future, once it's clearer about having a bot or in the case checks are enough.

Wdyt?

Reply:
SGTM